src/server/luaentity_sao.cpp

```diff
@@ -256,13 +256,13 @@ std::string LuaEntitySAO::getClientInitializationData(u16 protocol_version)
 	msg_os << serializeString32(getPropertyPacket()); // message 1
 	msg_os << serializeString32(generateUpdateArmorGroupsCommand()); // 2
 	msg_os << serializeString32(generateUpdateAnimationCommand()); // 3
-	for (const auto &bone_pos : m_bone_position) {
+	for (const auto &bone_pos : m_bone_override) {
```

Comment:
nitpick: get rid of `bone_pos` if you want to keep the naming consistent

Reply:
(C++-wise) kept only `generateUpdateBonePositionCommand` with `BonePosition` in its name (given that this method name is based on the historic packet name, I'm somewhat reluctant to rename that method).

Reply:
You can rename it if you want

Reply:
Alright, done https://github.com/minetest/minetest/pull/12388/commits/e55eafc18678e05ef9624cd216c85a704a940215.